src/pages/admin.vue

```diff
@@ -1,91 +1,29 @@
 <script setup lang="ts">
-import { ROLE } from "@/constants";
-import api, { fetcher } from "@/models/api";
 import { useSession } from "@/stores/session";
-import useVuelidate from "@vuelidate/core";
-import { required, maxLength, between, integer, minLength } from "@vuelidate/validators";
-import { useAxios } from "@vueuse/integrations/useAxios";
-import axios from "axios";
-import { computed, ref, watchEffect } from "vue";
+import { watchEffect } from "vue";
 import { useRoute, useRouter } from "vue-router";
 import { useI18n } from "vue-i18n";
+import { useStorage, useTitle } from "@vueuse/core";
+import { LOCAL_STORAGE_KEY } from "@/constants";
 
+useTitle("Admin | Normal OJ");
 const route = useRoute();
 const router = useRouter();
 const { t } = useI18n();
-const tabs = [{ text: t("admin.tab.user"), value: "user" }];
+const tab = useStorage(LOCAL_STORAGE_KEY.ADMIN_TAB, "/admin/user");
+const tabs = [{ text: t("admin.tab.user"), value: "/admin/user" }];
 watchEffect(() => {
-  if (!route.query.tab || !tabs.some((d) => d.value === route.query.tab)) {
-    router.push("/admin?tab=user");
+  if (!tabs.some((d) => d.value === route.path)) {
+    router.replace(tab.value);
```

Comment:
I think this part can be written in a middleware in the future. When there's more tabs on admin page

Reply:
sure, thanks for the suggestion!